Packs/Core/Playbooks/playbook-Uncommon_remote_scheduled_task_created.yml

```diff
@@ -0,0 +1,1352 @@
+id: Uncommon remote scheduled task created
+version: -1
+contentitemexportablefields:
+  contentitemfields: {}
+name: Uncommon remote scheduled task created
+description: "This playbook is designed to handle the alert \"Uncommon remote scheduled task created\".\n\nPlaybook Stages:\n\nAnalysis:\n\n- The playbook checks if the remote IP is external or has a bad reputation.\n\nInvestigation:\nDuring the alert investigation, the playbook will perform the following:\n\n- Searches for related XDR alerts to the endpoint using the following MITRE techniques to identify any malicious activity: T1202 - Indirect Command Execution, T1021 - Remote Services.\n- Searches for related XSIAM agent alerts on the remote endpoint to determine if the creation of the scheduled task is part of an attack pattern.\n- Searches for suspicious command-line parameters indicating a malicious scheduled task.\n\nRemediation:\n\n- Remediation actions will disable the scheduled task and block the malicious IP, and close the alert.\n\nRequirements:\n\nFor response actions, the following integrations are required: \n\n- PAN-OS."
```

Comment:
Recommend:
 "This playbook handles \"Uncommon remote scheduled task created\" alerts.\n\nPlaybook Stages:\n\nAnalysis:\n\n- The playbook checks if the remote IP is external or has a bad reputation.\n\nInvestigation:\nThe playbook searches for the following:\n\n- Related Cortex XDR alerts on the endpoint that use the following MITRE techniques to identify any malicious activity: T1202 - Indirect Command Execution, T1021 - Remote Services.\n- Related Cortex XSIAM agent alerts on the remote endpoint, to determine if the creation of the scheduled task is part of an attack pattern.\n- Suspicious command line parameters indicating a malicious scheduled task.\n\nRemediation:\n\n- Disable the scheduled task,  block the malicious IP, and close the alert.\n\nRequirements:\n\nFor response actions, the following integrations are required: \n\n- PAN-OS."